### PR TITLE
GIVCAMP-81 GIVCAMP-82 | HTML head and SEO

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Add SEO/HTML head to pages using the Gatsby Head API in v5

# Review By (Date)
- Retro

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to the preview homepage
2. The SEO field has been added to the homepage on Storyblok, inspect the code of the homepage and see that these items appear in the HTML head

Screenshot 2023-05-17 at 4 24 40 PM

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-81, GIVCAMP-82

### CODE REVIEW

File: src/pages/{storyblokEntry.full_slug}.tsx [hunk 1]
```diff
@@ -15,20 +16,34 @@ const StoryblokEntry: React.FC<PageProps<DataProps>> = ({
...
+  const blok = story.content;
+
+  return (
+    <PageHead
+      title={blok.title || story.name}
+      heroImage={blok.heroImage || blok.hero.image}
+      seo={blok.seo}
+    />
+  );
+};
```
Comment (yvonnetangsu) [inline src/pages/{storyblokEntry.full_slug}.tsx:45]: This is the new way of handling html head elements in Gatsby 5. I'm asking a question on Discord whether it's better to put the font stuff inside `PageHead`, or in the `gatsby-ssr`. Their official doc still says gatsby-ssr, but their document is all over the place at the moment.

File: src/components/PageHead/PageHead.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export const PageHead = (props: PageHeadProps) => (
+  <>
+    <html lang="en" />
+    <SEO {...props} />
+    <link rel="icon" type="image/png" href="https://www-media.stanford.edu/assets/favicon/favicon-192x192.png" sizes="192x192" />
+    <link rel="icon" type="image/png" sizes="180x180" href="https://www-media.stanford.edu/assets/favicon/apple-touch-icon-180x180.png" />
+    <link rel="icon" type="image/png" href="https://www-media.stanford.edu/assets/favicon/favicon-128.png" sizes="128x128" />
+    <link rel="icon" type="image/png" href="https://www-media.stanford.edu/assets/favicon/favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="https://www-media.stanford.edu/assets/favicon/favicon-16x16.png" sizes="16x16" />
```
Comment (yvonnetangsu) [inline src/components/PageHead/PageHead.tsx:14]: I trimmed down the list of favicons to this - I think some of the other sizes are deprecated, and I remove the IE only ones.

File: src/components/PageHead/PageHead.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { SEO, SEOProps } from './SEO';
+
+export type PageHeadProps = SEOProps;
+
+export const PageHead = (props: PageHeadProps) => (
+  <>
+    <html lang="en" />
```
Comment (yvonnetangsu) [inline src/components/PageHead/PageHead.tsx:8]: lang="en" can be added via the Gatsby Head also

**Comment by sherakama** [comment]
Yay using new APIs

**Pull request merged**